packages/common/src/hooks/useAuthContext.ts

```diff
@@ -0,0 +1,54 @@
+import { useEffect } from 'react';
+import { useLocalStorage } from '../localStorage';
+import { Store, User } from '../types';
+import { useHostContext } from './useHostContext';
+import Cookies from 'js-cookie';
+import { addMinutes } from 'date-fns';
+import { useOmSupplyApi } from '../api';
+
+const COOKIE_LIFETIME_MINUTES = 60;
+
+interface AuthCookie {
+  store?: Store;
+  token: string;
+  user?: User;
+}
+
+export const getAuthCookie = (): AuthCookie => {
+  const authString = Cookies.get('auth');
+  const emptyCookie = { token: '' };
+  if (!!authString) {
+    try {
+      const parsed = JSON.parse(authString) as AuthCookie;
+      return parsed;
+    } catch {
+      return emptyCookie;
+    }
+  }
+  return emptyCookie;
+};
+
+export const useAuthContext = () => {
```

Comment:
Calling this `AuthContext` confused me a bit - I was looking for the `Provider` component - maybe `useAuthState`?